common/source/docs/common-triducer.rst

```diff
@@ -0,0 +1,40 @@
+.. _common-triducer:
+
+[copywiki destination="copter,plane,rover"]
+
+=======================
+Triducer
+=======================
+
+Triducer combines Speed, temperature and Depth into one Transducer.
+
+Mounting the triducer
+----------------------
```

Comment:
should be check if the lenght it right, otherwise, it will trigger an annoying warning

Reply:
ok, I guess I correct that also.